homeassistant/components/unifiprotect/config_flow.py

```diff
@@ -97,7 +97,7 @@ async def async_step_discovery_confirm(
         placeholders = {
             "name": discovery_info["hostname"]
             or discovery_info["platform"]
-            or f"NVR {discovery_info['mac']}",
+            or f"NVR {_async_short_mac(discovery_info['mac'])}",
```

Comment:
I guess this is fine. What is nice about the IP address is that I could navigate to the IP address and see what device it is.

Reply:
Now maybe we should add a "visit device" option to discovered devices.

Reply:
Ah, I see now short mac instead of full mac 👍

Reply:
I'm hoping in the future we will be able to get the name of the device, but currently we need root to do it and have to do a very expensive `scapy` setup and teardown to make it happen.  We had hoped to be able to get it to give it up without a login, but the current only efficient way involves webscraping.  Still a few more days before beta though so we might find a way

Reply:
I think I actually just found away.

Probably a bump of `unifi-discovery` shortly if it works

Reply:
https://github.com/home-assistant/core/pull/64391